site/content/contribute/mobile/push-notifications/corporate-proxy.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Corporate Proxy Server"
+date: 2020-03-09T11:35:32
+weight: 3
+subsection: "Setup Push Notifications"
+---
+
+### How do I receive mobile push notification if my IT policy requires the use of a corporate proxy server?
+
+When your IT policy requires a corporate proxy to scan and audit all outbound traffic the following options are available:
+
+###### 1 - Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment
```

Comment:
```suggestion
###### Option 1: Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment
```

Reply:
afaik these are more three different situations then multiple options?

Reply:
We say `the following options are available:`, but I'm not sure we've used "Option X" format before. @justinegeffen wondering if we leave unchanged?

Reply:
Good question. I *think* I have seen this in other docs but can't link them off the top of my head and the style guide doesn't specify whether we use it or not. In this context it works and I'd say we can leave it unchanged.